src/proto/CMakeLists.txt

```diff
@@ -1,41 +1,59 @@
-set(OUTPUT_FILES
-        ${CMAKE_CURRENT_BINARY_DIR}/faabric.pb.cc
-        ${CMAKE_CURRENT_BINARY_DIR}/faabric.pb.h
-        ${CMAKE_CURRENT_BINARY_DIR}/faabric.grpc.pb.cc
-        ${CMAKE_CURRENT_BINARY_DIR}/faabric.grpc.pb.h
-        )
+# ----------------------------------------------
```

Comment:
Changes in this file are just a refactor and steps to copy the generated headers into place.